baseplate/lib/live_data/zookeeper.py

```diff
@@ -56,12 +59,21 @@ def zookeeper_client_from_config(
         credentials = secrets.get_simple(path)
         auth_data.append(("digest", credentials.decode("utf8")))
 
+    # Kazoo requires different parameters depending on whether
+    # we are using gevent or not.
+    if gevent_is_patched():
+        handler = SequentialGeventHandler()
+        sleep_func = gevent.sleep
+    else:
+        handler = None
+        sleep_func = time.sleep
```

Comment:
If gevent does it's [patching this becomes gevent.sleep anyway](https://www.gevent.org/api/gevent.monkey.html#gevent.monkey.patch_time). Do we need to explicitly use one or the other here or can we just use time.sleep always?

Reply:
You're right, this can be removed!

I added it because of a failing test, but in retrospect that was because the sleep function hadn't been patched in that test.